src/bootstrap/src/core/builder/tests.rs

```diff
@@ -1516,6 +1516,7 @@ mod snapshot {
                 .path("compiler")
                 .render_steps(), @r"
         [check] rustc 0 <host> -> rustc 1 <host>
+        [check] rustc 0 <host> -> rustc 1 <host>
```

Comment:
Question: this feels a bit surprising, but I guess this is a no-op since the check build artifact can be reused?